### ISSUE TITLE
Should we delete or archive this repo?

### ISSUE THREAD

**Issue by yeqbfgxjiq** [sourcecred/notes#3]
Seems like a lot of notes and discussions are happening on the [SourceCred Discourse](https://discourse.sourcecred.io/). To keep things in one place and make it easy for new people to contribute should we delete or archive this repo?

**Comment by teamdandelion** [comment]
Yes, this is definitely superceded by the Discourse. I will archive it.
